lib/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @typedef {import('unist').Point} Point
+ * @typedef {import('unist').Position} UnistPosition
+ * @typedef {import('vscode-languageserver-types').Position} LspPosition
+ * @typedef {import('vscode-languageserver-types').Range} Range
+ */
+
+/**
+ * Convert a unist point to an LSP position.
+ *
+ * @param {Point} point The unist point to convert.
+ * @returns {LspPosition} The point converted to an LSP position.
+ */
+export function unistPointToLspPosition(point) {
+  return {
+    character: point.column - 1,
+    line: point.line - 1
+  }
+}
+
+/**
+ * Convert an LSP position to a unist point.
+ *
+ * @param {LspPosition} position The LSP position to convert.
+ * @returns {Point} The position converted to a unist point.
+ */
+export function lspPositionToUnistPoint(position) {
+  return {
+    column: position.character + 1,
+    line: position.line + 1
+  }
+}
+
+/**
+ * Convert a unist position to an LSP range.
+ *
+ * @param {UnistPosition} position The unist position to convert.
+ * @returns {Range} The position converted to an LSP range.
+ */
+export function unistPositionToLspRange(position) {
+  return {
+    start: unistPointToLspPosition(position.start),
+    end: unistPointToLspPosition(position.end)
+  }
+}
```

Comment:
Currently `unified-language-server` [uses the `Position.create` and `Range.create` helper functions](https://github.com/unifiedjs/unified-language-server/blob/3.1.0/lib/index.js#L56-L100) from `vscode-languageserver-types`, but IMO we could just trust the unist positional information is valid.